.github/dependabot.yml

```diff
@@ -1,19 +1,19 @@
 version: 2
 updates:
-    # poetry is part of the pip ecosystem
-    - package-ecosystem: pip
-      directory: "/"
-      schedule:
-          interval: daily
-      open-pull-requests-limit: 10
-      ignore:
-          - dependency-name: "asyncpg"
-          - dependency-name: "flake8"
-            versions: [">=4.0.0"]
-            # flake8 >= 4.0 conflicts with importlib_metadata > 4.3 as per this: PyCQA/flake8#1438
-          - dependency-name: "sphinx tabs"
-            versions: [">=3.4.0"]
-            # sphinx tabs >=3.3.1 requires docutils > 0.18 and sphinx rtd theme is not compatible with docutils >= 0.18
-          - dependency-name: "Sphinx"
-            versions: [">=4.6.0"]
-            # sphinx-tabs (3.3.1) depends on sphinx (>=2,<5)
+# poetry is part of the pip ecosystem
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+  ignore:
+   - dependency-name: "asyncpg"
+   - dependency-name: "flake8"
+     versions: [">=4.0.0"]
+     # flake8 >= 4.0 conflicts with importlib_metadata > 4.3 as per this: PyCQA/flake8#1438
+   - dependency-name: "sphinx tabs"
+     versions: [">=3.4.0"]
+     # sphinx tabs >=3.3.1 requires docutils > 0.18 and sphinx rtd theme is not compatible with docutils >= 0.18
+   - dependency-name: "Sphinx"
+     versions: [">=4.6.0"]
+     # sphinx-tabs (3.3.1) depends on sphinx (>=2,<5)      
```

Comment:
I wonder if this one is even required. Perhaps the reason it took so long for dependabot to open the bump PR for sphinx is that the sphinx-tabs constraint was not compatible. It doesn't hurt to have it here though.